build/Helix/GenerateTestProjFile.ps1

```diff
@@ -0,0 +1,304 @@
+Param(
+    [Parameter(Mandatory = $true)] 
+    [string]$TestFile,
+
+    [Parameter(Mandatory = $true)] 
+    [string]$OutputProjFile,
+
+    [Parameter(Mandatory = $true)] 
+    [string]$JobTestSuiteName,
+
+    [Parameter(Mandatory = $true)] 
+    [string]$TaefPath
+)
+
+Class TestCollection
+{
```

Comment:
Neat! I actually didn't know you could do structured classes in Powershell, I always use PSCustomObject. But this is cool!

Reply:
Yup - it's a new feature as of PowerShell 5. :)